app/serializers/api/admin/variant_serializer.rb

```diff
@@ -5,11 +5,9 @@ module Admin
     class VariantSerializer < ActiveModel::Serializer
       attributes :id, :name, :producer_name, :image, :sku, :import_date,
                  :options_text, :unit_value, :unit_description, :unit_to_display,
-                 :display_as, :display_name, :name_to_display,
+                 :display_as, :display_name, :name_to_display, :variant_overrides_count,
                  :price, :on_demand, :on_hand, :in_stock, :stock_location_id, :stock_location_name
 
-      has_many :variant_overrides
```

Comment:
tests are passing without this so we might be lucky :pray:

Reply:
It's only used in the bulk products pages, and they don't use overrides :+1: